v2/internal/testcommon/kube_test_context_envtest.go

```diff
@@ -189,19 +189,21 @@ func createSharedEnvTest(cfg testConfig, namespaceResources *namespaceResources)
 		log.Println("Waiting for webhook server to start")
 		// Need to block here until things are actually running
 		chk := mgr.GetWebhookServer().StartedChecker()
-		timeoutAt := time.Now().Add(5 * time.Second)
+		timeoutAt := time.Now().Add(15 * time.Second)
 		for {
 			err = chk(nil)
 			if err == nil {
 				break
 			}
 
 			if time.Now().After(timeoutAt) {
-				panic("timed out waiting for webhook server to start")
+				err = errors.Wrap(err, "timed out waiting for webhook server to start")
+				panic(err)
```

Comment:
I think just reporting the error message rather than the full dump of the error struct would be better here.
```suggestion
				panic(err.Error())
```